CI-NightlyBuilds.md

```diff
@@ -0,0 +1,108 @@
+# Ice Nightly Builds
+
+Nightly builds of Ice are generated by GitHub Actions from the main development branch. These builds include all
+packages intended for the upcoming 3.8 release.
```

Comment:
Should this be "for the next non-patch Ice release"?

I assume we'll keep these Nightly builds for main once Ice 3.8 is released and 3.8 gets its own branch.